File: PlayerBuild/SplashStats.cs
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TT2Advisor.Import;

namespace TT2Advisor.PlayerBuild
{
    public class SplashStats
    {
        //public SplashStats(JToken splashStatsToken)
        //{
        //    SplashDamage = (double)splashStatsToken.SelectToken("['Splash Damage']");
        //    BaseMaxSplashCount = (int)splashStatsToken.SelectToken("['Base Max Splash Count']");
        //    AllSplashSkip = (decimal)splashStatsToken.SelectToken("['All Splash Skip']");
        //    HeavenlyStrikeSplashCount = (int)splashStatsToken.SelectToken("['Heavenly Strike Splash Count']");
        //    LightningBurstSplashCount = (int)splashStatsToken.SelectToken("['Lightning Burst Splash Count']");
        //    ClanShipSplashCount = (int)splashStatsToken.SelectToken("['Clan Ship Splash Count']");
        //    DaggerSplashCount = (int)splashStatsToken.SelectToken("['Dagger Splash Count']");
        //    HeavenlyStrikeSplashSkip = (decimal)splashStatsToken.SelectToken("['Heavenly Strike Splash Skip']");
        //    PetSplashSkip = (decimal)splashStatsToken.SelectToken("['Pet Splash Skip']");
        //    ClanShipSplashSkip = (decimal)splashStatsToken.SelectToken("['Clan Ship Splash Skip']");
        //    ShadowCloneSplashSkip = (decimal)splashStatsToken.SelectToken("['Shadow Clone Splash Skip']");
        //    DaggerSplashSkip = (decimal)splashStatsToken.SelectToken("['Dagger Splash Skip']");
        //}

        public SplashStats(Import.SplashStats splashStats)
        {
            SplashDamage = splashStats.SplashDamage.ToDouble();
            BaseMaxSplashCount = splashStats.BaseMaxSplashCount.ToInt();
            AllSplashSkip = splashStats.AllSplashSkip.ToDecimal();
            HeavenlyStrikeSplashCount = splashStats.HeavenlyStrikeSplashCount.ToInt();
            LightningBurstSplashCount = splashStats.LightningBurstSplashCount.ToInt();
            ClanShipSplashCount = splashStats.ClanShipSplashCount.ToInt();
            DaggerSplashCount = splashStats.DaggerSplashCount.ToInt();
            HeavenlyStrikeSplashSkip = splashStats.HeavenlyStrikeSplashSkip.ToDecimal();
            PetSplashSkip = splashStats.PetSplashSkip.ToDecimal();
            ClanShipSplashSkip = splashStats.ClanShipSplashSkip.ToDecimal();
            ShadowCloneSplashSkip = splashStats.ShadowCloneSplashSkip.ToDecimal();
            DaggerSplashSkip = splashStats.DaggerSplashSkip.ToDecimal();
        }

        public double SplashDamage { get; set; }
        public int BaseMaxSplashCount { get; set; }
        public decimal AllSplashSkip { get; set; }
        public int HeavenlyStrikeSplashCount { get; set; }
        public int LightningBurstSplashCount { get; set; }
        public int ClanShipSplashCount { get; set; }
        public int DaggerSplashCount { get; set; }
        public decimal HeavenlyStrikeSplashSkip { get; set; }
        public decimal PetSplashSkip { get; set; }
        public decimal ClanShipSplashSkip { get; set; }
        public decimal ShadowCloneSplashSkip { get; set; }
        public decimal DaggerSplashSkip { get; set; }
    }
}

File: TT2Advisor.UI/ClientApp/src/components/myBuild/splashStats.js
import React, { Component } from 'react';

export class SplashStats extends Component {

    splashStats: any;

    constructor(props) {
        super(props);
        this.state = { splashStats: this.props.splashStats, loading: true };
    }

    componentDidMount() {
        
    }

    render() {
        console.log(this.state.splashStats);
        return (
            <div>
                <h2>Splash Stats</h2>
                <table className='table table-striped' aria-labelledby="tabelLabel">
                    <thead>
                        <tr>
                            <th>Stat</th>
                            <th>Value</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Splash Damage</td>
                            <td>{this.state.splashStats.splashDamage}</td>
                        </tr>
                        <tr>
                            <td>Base Max Splash Count</td>
                            <td>{this.state.splashStats.baseMaxSplashCount}</td>
                        </tr>
                        <tr>
                            <td>All Splash Skip</td>
                            <td>{this.state.splashStats.allSplashSkip}</td>
                        </tr>
                        <tr>
                            <td>Heavenly Strike Splash Count</td>
                            <td>{this.state.splashStats.heavenlyStrikeSplashCount}</td>
                        </tr>
                        <tr>
                            <td>Lightning Burst Splash Count</td>
                            <td>{this.state.splashStats.lightningBurstSplashCount}</td>
                        </tr>
                        <tr>
                            <td>Clan Ship Splash Count</td>
                            <td>{this.state.splashStats.clanShipSplashCount}</td>
                        </tr>
                        <tr>
                            <td>Dagger Splash Count</td>
                            <td>{this.state.splashStats.daggerSplashCount}</td>
                        </tr>
                        <tr>
                            <td>Heavenly Strike Splash Skip</td>
                            <td>{this.state.splashStats.heavenlyStrikeSplashSkip}</td>
                        </tr>
                        <tr>
                            <td>Pet Splash Skip</td>
                            <td>{this.state.splashStats.petSplashSkip}</td>
                        </tr>
                        <tr>
                            <td>Clan Ship Splash Skip</td>
                            <td>{this.state.splashStats.clanShipSplashSkip}</td>
                        </tr>
                        <tr>
                            <td>Shadow Clone Splash Skip</td>
                            <td>{this.state.splashStats.shadowCloneSplashSkip}</td>
                        </tr>
                        <tr>
                            <td>Dagger Splash Skip</td>
                            <td>{this.state.splashStats.daggerSplashSkip}</td>
                        </tr>
                    </tbody>
                </table>
            </div>
        );
    }
}
